strawberryfields/gbs/similarity.py

```diff
@@ -95,62 +138,17 @@ def sample_to_event(sample: list, max_count_per_mode: int) -> Union[int, None]:
             attributed the event ``None``.
 
     Returns:
-        list[int]: the orbit of the sample
+        int or None: the event of the sample
     """
     if max(sample) <= max_count_per_mode:
         return sum(sample)
 
     return None
 
 
-def orbits(photon_number: int) -> Generator[list, None, None]:
-    """Generate all the possible orbits for a given photon number.
-
-    Provides a generator over the integer partitions of ``photon_number``.
-    Code derived from `website <http://jeromekelleher.net/generating-integer-partitions.html>`__
-    of Jerome Kelleher's, which is based upon an algorithm from Ref. :cite:`kelleher2009generating`.
-
-    **Example usage:**
-
-    >>> o = orbits(5)
-    >>> list(o)
-    [[1, 1, 1, 1, 1], [2, 1, 1, 1], [3, 1, 1], [2, 2, 1], [4, 1], [3, 2], [5]]
-
-    Args:
-        photon_number (int): number of photons to generate orbits from
-
-    Returns:
-        Generator[list[int]]: orbits with total photon number adding up to ``photon_number``
-    """
-    a = [0] * (photon_number + 1)
-    k = 1
-    y = photon_number - 1
-    while k != 0:
-        x = a[k - 1] + 1
-        k -= 1
-        while 2 * x <= y:
-            a[k] = x
-            y -= x
-            k += 1
-        l = k + 1
-        while x <= y:
-            a[k] = x
-            a[l] = y
-            yield sorted(a[: k + 2], reverse=True)
-            x += 1
-            y -= 1
-        a[k] = x + y
-        y = x + y - 1
-        yield sorted(a[: k + 1], reverse=True)
-
-

```

Comment:
This was just moved down below, as I felt the order reported in the doc's "code summary" was better like this.